File: backend/src/main/java/vr/balance/app/API/request/CompletedBalanceTestExerciseDTO.java
package vr.balance.app.API.request;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.SuperBuilder;
import vr.balance.app.API.request.base.CompletedExerciseDTO;

@EqualsAndHashCode(callSuper = true)
@Data
@SuperBuilder
public class CompletedBalanceTestExerciseDTO extends CompletedExerciseDTO {
    String phase_1;
    String phase_2;
    String phase_3;
    String phase_4;
}

File: Assets/Scripts/Models/Request/Exercise/CompletedBalanceTestExerciseDTO.cs
using System.Collections.Generic;
using UnityEngine;

namespace Models.DTO.Exercise
{
    public class CompletedBalanceTestExerciseDTO : CompletedExerciseDTO
    {
        public List<Vector3DTO> phase_1;
        public List<Vector3DTO> phase_2;
        public List<Vector3DTO> phase_3;
        public List<Vector3DTO> phase_4;
    }
}